Slims/slims.py

```diff
@@ -0,0 +1,183 @@
+import os
+import re
+
+from slims.slims import Slims
+from slims.criteria import is_one_of, equals, conjunction, not_equals
+from slims.content import Status
+
+
+
+class slims:
+    url = os.environ.get('SLIMS_URL')
+    user = os.environ.get('SLIMS_USER')
+    password = os.environ.get('SLIMS_PASSWORD')
+
+
+instance = 'wgs-somatic_query'
+url = slims.url
+user = slims.user
+password = slims.password
+slims = Slims(instance, url, user, password)
```

Comment:
Variable name could clash with slims python api. Consider renaming it to something like `slims_connection`.